src/signals/incident-management/containers/IncidentDetail/components/DetailHeader/components/DownloadButton/index.js

```diff
@@ -25,6 +23,7 @@ class DownloadButton extends React.Component {
       responseType: 'blob'
     }).then((response) => response.blob())
         .then((blob) => {
+          /* istanbul ignore next */
```

Comment:
Why ignore the following line? You can mock the navigator and have it covered that way.

Reply:
het lukt om te Stubben:
`window.navigator.msSaveOrOpenBlob = jest.fn();`

maar als ik probeer te kijken of hij aangeroepen wordt, dan niet:
`expect(window.navigator.msSaveOrOpenBlob).toHaveBeenCalled();`

Expected mock function to have been called, but it was not called.

hij voert wel de code uit.